Assignment 5.Rmd

```diff
@@ -60,18 +72,34 @@ plot(score_ctree)
 
 Please interpret the tree, which two behaviors do you think the teacher should most closely pay attemtion to?
 
+Teachers should pay attention to hint and prior_prob_count. Students who has asked for hints are generally more likely to 
```

Comment:
Good answer, you could improve it by referring directly to the tree you created.